railties/test/commands/server_test.rb

```diff
@@ -44,6 +44,22 @@ def test_environment_with_rack_env
     end
   end
 
+  def test_caching_without_option
+    args = []
+    options = Rails::Server::Options.new.parse!(args)
+    assert_equal nil, options[:caching]
+  end
+
+  def test_caching_with_option
+    args = ["-C"]
```

Comment:
This could potentially cause some confusion, as there is already a `-c` option. I think it should be changed, I just didn't have a notion of what it should be.